vpr/src/base/load_flat_place.h

```diff
@@ -0,0 +1,16 @@
+#ifndef LOAD_FLAT_PLACE_H
+#define LOAD_FLAT_PLACE_H
+
+#include "vpr_types.h"
+
+/**
+ * @brief A function that prints a flat placement file
+ */
+void print_flat_placement(const char* flat_place_file);
+
+/**
+ * @brief A function that loads and legalizes a flat placement file
```

Comment:
It would be good to briefly explain here or point at to a high-level comment elsewhere for what a flat placement is (an x,y,sub_tile and primitive_index_in_cluster) file that implies both a packing and a placement).

Reply:
layer is for 3D devices; when you implement this function you an assert that there is only one layer for now if you don't support multi-layer devices (and you probably don't). The grid data structure knows how many layers there are.

Reply:
Should comment the parameters; I'm not sure of what exactly archi_id is. vpr_setup is pretty clear but still good to have a basic comment for consistency.

Reply:
I've added this definition in comments and will also include in the documentation.

I've added a check for num layers > 1 in vpr_api just before attempting to load the flat placement.

arch_id was a placeholder (to avoid a CI failure for unused args); it is now t_arch& arch.  I added a new function, vpr_load_flat_place(vpr_setup, arch).  These args are the same as those for vpr_pack_flow, vpr_place_flow, and vpr_route_flow.